force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +49,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        Integer eligibleToVoteAge = 18;
+        Boolean eligibleToVote;
+
+        eligibleToVote = age >= eligibleToVoteAge? true : false;
+            return eligibleToVote; // Replace null with the variable you used to store the result
```

Comment:
Good work. You could just return after evaluating the age and skipped the variable assignment.  When you evaluate with  operators like < > or = they evaluate to a Boolean. 

  public static Boolean votingEligibility(Integer age) {
        return age >= 18;
    }